### ISSUE TITLE
Extension doesn't seem to work

### ISSUE THREAD

**Issue by mjeshurun** [katagaki/Unagent#25]
Hi, 
I tried using the extension on Safari running on iOS 17.5.1, but when I tried to set the user agent to a Chrome, Safari or Edge browser running on MacOS, the mobile sites I tested still showed me the mobile versions of the website. 
I was also directed to download their app, instead of letting me login on the browser.

**Comment by katagaki** [comment]
Hi @mjeshurun 

Are you able to provide a sample page where the issue is happening?

It could be due to the website detecting the viewport (screen) size, similar to what Apple does for App Store Connect does.

**Comment by Happyfeet01** [comment]
Not working anymore with safari on iOS.

**Comment by katagaki** [comment]
Hi @Happyfeet01 

Could you provide the below information for further investigation?

- iOS version
- Unagent global user agent set
- Unagent per site settings set
- Screenshot, if possible, of Settings > Safari > Extensions > Unagent
- Screenshot, if possible, of the ‘Aa’ menu in Safari
- Current version of Unagent (you can check this by going into Settings > General > iPad Storage > Unagent)

**Comment by Happyfeet01** [comment]
Hi, 

I have iOS 17.6 installed.

Here are the settings for safari 

Useragent (but no difference between others)

`Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.6422.113 Mobile Safari/537.36`

![image](https://github.com/user-attachments/assets/083b6dc1-b788-4b26-b264-2186cfd8bbcd)
![image](https://github.com/user-attachments/assets/51842806-5d77-47ec-a8b6-6199d94dbb19)

![image](https://github.com/user-attachments/assets/28f6bcf7-014f-4c77-af84-f7dc2c692554)

![image](https://github.com/user-attachments/assets/a135f454-38ea-4971-ab6c-8c78769f32f7)

**Comment by katagaki** [comment]
@Happyfeet01 

Could you confirm if the user agent you set is for the **Global User Agent** or a **Per-Site Setting** for the website in the screenshot you sent?

If you can, set a **Per-Site Setting** for `www.whatsmyua.info` and navigate to the below site in Safari, and let me know the results.
[https://www.whatsmyua.info](https://www.whatsmyua.info)

**Comment by katagaki** [comment]
Closing this issue for the time being, as the mentioned issues cannot be reproduced.

As a fallback, I have implemented a Troubleshooting page in the Safari extension, which can be accessed by tapping on the Unagent icon (macOS) or menu item (iOS). There is an option to force reset all user agent rules in that page which lets you force reset all rules and start afresh.

For some reason, user agent rules may not be reliably erased, even when Unagent is reinstalled, which is causing some of these issues.

Update to the Unagent 1.6 and above to use the Troubleshooting page!

[App Store](https://apps.apple.com/app/id6448220941)
